### PR TITLE
style: Rename django-python-hello-world samples to be unique

### PR DESCRIPTION
**Pull request by SKrupa**
Currently they use the same name as the existing python-hello-world sample. As these samples are deploying different code, they should use different names. This is consistent with how the guestbook samples are defined, where the django sample use a different name than the base python ones.

### CODE REVIEW

File: python/django/python-hello-world/kubernetes-manifests/hello.deployment.yaml [hunk 1]
```diff
@@ -6,25 +6,25 @@
 apiVersion: apps/v1
```
Comment (averikitsch) [inline python/django/python-hello-world/kubernetes-manifests/hello.deployment.yaml:6]: Missing update on line 2

**Review by averikitsch — CHANGES_REQUESTED** [review]

File: python/django/python-hello-world/kubernetes-manifests/hello.deployment.yaml (hunk 1 above)
Comment (SKrupa) [reply python/django/python-hello-world/kubernetes-manifests/hello.deployment.yaml:6]: whoops, fixed. Thanks!

**Review by averikitsch — APPROVED** [review]

**Pull request merged**